PlayRho/Common/Math.hpp

```diff
@@ -392,9 +392,6 @@ PLAYRHO_CONSTEXPR inline auto Dot(const T1 a, const T2 b) noexcept
 ///
 /// @sa https://en.wikipedia.org/wiki/Cross_product
 ///
-/// @param a Value A of a 2-element type.
```

Comment:
Hmm... did I make this change? LOLs! Wondering now why I did this. I'm imagining this relates to the doxygen build check that's run in CI. But I'd think removing these doxygen comments would cause this part of the build to have an issue instead of fixing an issue with it. I guess it can always be added back in later easily enough if needed.